src/app/components/ImageCard/ImageCard.tsx

```diff
@@ -0,0 +1,65 @@
+import Image from 'next/image';
+import { type CSSProperties, memo } from 'react';
+import styles from './ImageCard.module.scss';
+import classNames from 'classnames';
+
+interface ImageCardProps {
+  imageUrl: string;
+  imageAlt: string;
+  children: React.ReactNode;
+  imageFirst?: boolean;
+  imageWidth?: number;
+  imageHeight?: number;
+  imageStyle?: CSSProperties;
+}
+
+const baseClasses = {
+  container:
+    'grid grid-cols-2 justify-items-center gap-4 xl:justify-items-start xl:gap-14',
```

Comment:
Also, I believe `grid-cols-2` is meant to be `grid-cols-1`. I noticed I was getting a new issue when shrinking my screen size.
<img width="601" alt="image" src="https://github.com/user-attachments/assets/9812ac02-0d09-4d74-8420-645c643df5f3">